tutorials/data-lake-hdlfs-dbx-connection/data-lake-hdlfs-dbx-connection.md

```diff
@@ -0,0 +1,116 @@
+
+## Details
+### You will learn
+  - How to setup a HANA Data Lake file container connection in the Database Explorer.
+  - Use the HDLFSCLI to add, remove and manage data files into the File Container.
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Collect the Required Connection Details)]
+
+To get the REST API Endpoint, login to the BTP Cockpit, click on the **Actions** tab of your Data Lake instance and then select **Open in SAP HANA Cloud Central**
+
+![Open in SAP HANA Cloud Central](image-1.png)
+
+Once you are in the Cloud Central, In the **Actions** of your Data Lake Instance menu select the **Copy Files REST API Endpoint** option.
+
+![Copy Files REST API Endpoint](image-2.png)
+
+The REST API Endpoint will be copied to the clipboard. It has a format like this:  
+
+`2db5a1f3-7e9a-4487-914b-8ce149bcfaac.files.hdl.prod-us10.hanacloud.ondemand.com`
+
+Make sure that your REST API endpoint has **files** keyword in it.
+
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Generating Client Certificate & Client Key )]
+
+
+You may already have these files available from configuring the Trust and Authorization in your HDLFS instance. If not, then you can regenerate them from the `ca.crt` that is configured in your HDLFS instance.  
+
+Follow the Step 3 and Step 4 from [Getting Started with Data Lake Files HDLFSCLI | Tutorials](data-lake-file-containers-hdlfscli) for SAP Developers exactly as it is.
+
+[DONE]
+[ACCORDION-END]
+
+
+[ACCORDION-BEGIN [Step 3: ](Adding the connection in Database Explorer)]
+
+Go to the Database explorer from BTP Cockpit or Cloud Central by clicking on the **Actions** menu of your Data Lake instance.
+
+Once you are in the Database explorer, click on the **+** symbol on top left panel. See Below.
+
+![Add Connection](image-3.png)
+
+A new window will pop up once. In the **Database Type** drop down, select type **Data Lake Files**. See Below.
+
+![ADD Database Type](image-4.png)
+
+In the next window, you will need to paste the **REST API Endpoint** that you copied earlier from the Cloud Central.
+
+Next is to upload the Client Certificate and Client Key (which will be created after completing the Getting Started with Data Lake Files HDLFSCLI | Tutorials for SAP Developers tutorial)
+
+Click on the **Upload file** button and browse to the location where you have saved the **client.crt** and **client.key** and upload them. You can give any name under **Display Name**
+
+![Upload Certificates](image-5.png)
+
+Once you click on OK. A **Data Lake File Store/Container** connection would be visible in the Databases section on the left in the Database Explorer. See Below.
+
+![Data Lake File Container](image-6.png)
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 4: ](Verify the File container connection)]
+
+We need to verify whether we have added the right connection of File Container in DBX. We can do that by adding/uploading a file to the File container and checking it in the DBX.
```

Comment:
Please don't use the "royal we" when addressing the user. Use "you" instead.
Please check the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9#mcetoc_1dtvf24ta2) for details.